### DISCUSSION TITLE
在最近自动更新zotero7.0 beta版本后，加载了style插件导致目录下不显示文献

### DISCUSSION THREAD

**Discussion by Danielsolucky** [MuiseDestiny/zotero-style#844] [Q&A]
在最近自动更新zotero7.0 beta版本后，加载了style插件，软件的文献目录就不显示任何文献了，重启软件后显示，但是一点击其他目录又不现实了，反复如此，关闭style之后重启才能恢复，意味着style不能用了

**Comment by wsczm1111** [comment]
我也遇到了一样的问题，应该是最新版style的问题，我用3.3.6没问题
